source/Outfit.h

```diff
@@ -143,6 +146,8 @@ class Outfit : public Weapon {
 	std::map<const Sound *, int> jumpSounds;
 	std::map<const Sound *, int> jumpInSounds;
 	std::map<const Sound *, int> jumpOutSounds;
+	std::map<const Sound *, int> cargoScanSounds;
+	std::map<const Sound *, int> outfitScanSounds;
```

Comment:
Is the int used for anything? If not, these can just be a std::vector

Reply:
I think hyperdrive sounds are handled in a similar way. So, if the ints aren't required, they can be removed there too.

Reply:
just read the comment above

Reply:
~~yeah, but I think it may not really work like this~~

Reply:
I can open a refactor to change these maps to some other containers once this PR is merged.

Reply:
what do you think would be best to use here, an `std::set`?